### DISCUSSION TITLE
Thanks to Oliver

### DISCUSSION THREAD

**Discussion by ramyaragupathy** [maplibre/maplibre#414] [Announcements]
As announced in the [blog](https://maplibre.org/news/2024-10-11-farewell-oliver/), Oliver will be stepping down from coordinator role this October 2024.  Oliver has ben the coordinator since the inception of MapLibre and has helped shape the project into what it is today. We would love for all you to share your personal message of thanks and well wishes for @wipfli here in this thread! 🙌

**Comment by davidoesch** [comment]
Hey Oli, man sieht sich immer zweimal im Leben--- vor allem so ein geospatial enthusiast wie du verliere ich nicht aus den Augen

All the Best 

Dave

**Comment by meenaxc** [comment]
Thanks Oliver for being a member of the founding team of MapLibre!

**Comment by Gugustinette** [comment]
Thanks Oliver 💙

**Comment by stefankarschti** [comment]
Thanks Oliver!

**Comment by sulewicz** [comment]
Thank you Oliver for all your hard work!

**Comment by dayjournal** [comment]
Thank you Oliver. 👍️
Your contributions to MapLibre have been wonderful. The MapLibre User Group Japan (MUG-JP) exists because I met you at FOSS4G 2022 Firenze.
I support your next challenge!!

Yasunori Kirimoto
